src/main/java/com/onaio/steps/handler/actions/ImportHandler.java

```diff
@@ -69,10 +71,16 @@ public boolean shouldOpen(int menu_id) {
     @Override
     public boolean open() {
         String deviceId = KeyValueStoreFactory.instance(activity).getString(HH_PHONE_ID);
-        String filename = Environment.getExternalStorageDirectory()+"/"+Constants.APP_DIR+"/"+Constants.EXPORT_FILE_NAME+"_"+
+        String surveyId = KeyValueStoreFactory.instance(activity).getString(HH_SURVEY_ID);
```

Comment:
since this value may change in the future to something else, could this be something that is configurable?